src/pages/home/components/Sidebar/components/SocialMediaForm/SocialMediaList.module.scss

```diff
@@ -0,0 +1,58 @@
+@use '~styles/global.scss' as *;
+@use '~styles/breakpoints.scss' as breakpoints;
+
+.selection {
+  display: grid;
+  flex-wrap: wrap;
+  grid-template-columns: repeat(auto-fit, minmax(165px, 1fr));
+  gap: 1.6rem;
+
+  .socialItem {
+    display: flex;
+    flex-direction: column;
+
+    align-items: center;
+    justify-content: center;
+
+    color: $primaryGray;
+    font-weight: bold;
+
+    padding: 1.2rem;
+    border: 1px solid $primaryGray;
+
+    transition: 0.09s ease-in-out;
+
+    cursor: pointer;
+
+    &:hover {
+      scale: 0.98;
+    }
+
+    p {
+      font-weight: 700;
+
+      margin-bottom: 2rem;
+    }
+
+    .button {
+      display: block;
+
+      font-size: 1.4rem;
+
+      margin-top: 1rem;
+
+      padding: 1rem 3rem;
+    }
+
+    > img {
```

Comment:
Não usar tag diretamente, sempre suar classes

Reply:
fix: change for class